PWGJE/EMCALJetTasks/CMakeLists.txt

```diff
@@ -36,6 +36,7 @@ include_directories(${ROOT_INCLUDE_DIRS}
                     ${AliPhysics_SOURCE_DIR}/PWG/CaloTrackCorrBase
                     ${AliPhysics_SOURCE_DIR}/PWGHF/jetsHF
                     ${AliPhysics_SOURCE_DIR}/PWGHF/vertexingHF
+                    ${AliPhysics_SOURCE_DIR}/PWGPP
```

Comment:
Since I'm not familiar with PID analyses are run, what functionality are you getting out of PWGPP? It's generally strongly discouraged to have PWG directories depend on each other since this can become a mess of circular dependencies. Obviously, we have to do this sometimes - and that's fine if so - but it would better if it can be avoided

Reply:
Hi Raymond,

Sorry, I wasn't aware that this was discouraged, but it makes sense. I was using this library for a filtered tree task and some ESD tools, but these aren't necessary for what I'm doing now. I've modified my code to not use any PWGPP dependencies. I'll close this pull request and open a new one now with the updated code.